### ISSUE TITLE
Program no longer compiles, and no errors as to why

### ISSUE THREAD

**Issue by Bippym** [prb28/vscode-amiga-assembly#233]
So,

I have updated to the latest version after a few months away. Now my program wont compile. I get a warning about auto-aligning an instruction, and that's is as far as it goes.

<deleted code now the issue is sorted>

**Comment by prb28** [comment]
Maybe the new vasm is the problem.
You may try with an older one.

I'll try to build your project next week.

**Comment by Real-MJoe** [comment]
prb28 is right. The new version of vasm spits out more warnings. But these bring the build process of the extension to a halt. Here the extension should give the possibility, if only warnings are present, to regard the whole thing as "Okay" anyway. But now to the simple solution:
The incude Sprite_Functions.asm has an odd number of bytes. Consequently the "Mouse.asm" ended up in an "odd" address. 
You can "easily" fix it by writing "EVEN" as the first command in the Mouse.asm. 
But there is also the possibility to work with the parameter "-align" (http://sun.hasenbraten.de/vasm/release/vasm_4.html). Then you have to adjust a few places.  For example in the "Sprite_Functions" line 396, there is also an EVEN missing.

Translated with www.DeepL.com/Translator (free version)

**Comment by prb28** [comment]
You are right, warnings should not stop a build, it should be optional.
I'll add a fix.

**Comment by prb28** [comment]
If you display the terminal, you'll see the warning during the build:
```
warning 51 in line 5 of "c:\Users\paulr\workspace\amiga\projects\BippyM\Overflow-22\src\Mouse.asm": instruction has been auto-aligned
        included from line 36 of "c:\Users\paulr\workspace\amiga\projects\BippyM\Overflow-22\Overflow.s"
>             move.w    #0,mouse_x
```

**Comment by Bippym** [comment]
> prb28 is right. The new version of vasm spits out more warnings. But these bring the build process of the extension to a halt. Here the extension should give the possibility, if only warnings are present, to regard the whole thing as "Okay" anyway. But now to the simple solution: The incude Sprite_Functions.asm has an odd number of bytes. Consequently the "Mouse.asm" ended up in an "odd" address. You can "easily" fix it by writing "EVEN" as the first command in the Mouse.asm. But there is also the possibility to work with the parameter "-align" (http://sun.hasenbraten.de/vasm/release/vasm_4.html). Then you have to adjust a few places. For example in the "Sprite_Functions" line 396, there is also an EVEN missing.

Thanks, I have added "even" to the end of all declarations, especially byte sized declarations. That has fixed it and it now compiles again thanks :)

**Comment by prb28** [comment]
published in pre-release 1.7.1

**Comment by Real-MJoe** [comment]
Nice one ,thx :)
![grafik](https://user-images.githubusercontent.com/40276633/183437810-30ad5282-7213-41c2-96d9-f01cb6163f83.png)
